packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -24,8 +25,8 @@ export class WallectConnectSession {
 	private readonly algodClient: algosdk.Algodv2;
 	wcAccounts: string[];
 
-	constructor(chain: string, connector?: WalletConnect) {
-		this.algodClient = algoexplorerAlgod(chain);
+	constructor(walletURL: HttpNetworkConfig, connector?: WalletConnect) {
```

Comment:
Should we check walletURL is valid or invalid?

Reply:
I think this is optional depending on how the user type the URL for the wallet.